packages/catalog-client/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@backstage/catalog-client",
   "description": "An isomorphic client for the catalog backend",
   "version": "0.5.0",
-  "main": "src/index.ts",
-  "types": "src/index.ts",
+  "main": "dist/index.cjs.js",
+  "types": "dist/index.d.ts",
```

Comment:
what are those changes required for?

Reply:
I didn't make these changes intentionally, so maybe it's part of some linting/cleaning process.

Regardless, I'm changing it to what's in master. I'm a little worried that this didn't happen during rebase.